landing_page/templates/landing_page/index.html

```diff
@@ -1,31 +1,126 @@
-{% extends 'landing_page/base.html' %}
-{% load static %}
+{% extends 'landing_page/base.html' %} {% load static %} {% block content %}
 
+<div id="message_div">
+  <h1>Welcome to ReeldIn</h1>
+</div>
+<div id="content_div">
+  <div id="title_div">
+    <h2>Latest Recommendations...</h2>
+    <button id="get_recommendations_btm">Get Recommendations</button>
+  </div>
+  <div class="movies_div content_container">
+    <div class="movie tooltip">
+      <div class="tooltiptext">
+        <h3>Spiderman: Into the Spiderverse</h3>
+        <p>Rating: 4.5/5</p>
+        <p>Genre: Animation, Action, Adventure</p>
+        <p>Year: 2018</p>
+        <p>Director: Bob Persichetti, Peter Ramsey, Rodney Rothman</p>
+        <button>Click</button>
+      </div>
 
-{% block content %}
+      <img
+        src="{% static 'landing_page/images/spiderverse.webp' %}"
+        alt="Movie 1"
+      />
+    </div>
 
-<h1>Welcome to the landing page</h1>
+    <div class="movie tooltip">
+      <div class="tooltiptext">
+        <h3>Spiderman: Into the Spiderverse</h3>
+        <p>Rating: 4.5/5</p>
+        <p>Genre: Animation, Action, Adventure</p>
+        <p>Year: 2018</p>
+        <p>Director: Bob Persichetti, Peter Ramsey, Rodney Rothman</p>
+        <button>Click</button>
+      </div>
 
-This is a blank landing page which inherits from landing_page/base.html. Feel free to add more routes to this app and edit the base.html file to add more content to the landing page.
+      <img
+        src="{% static 'landing_page/images/spiderverse.webp' %}"
+        alt="Movie 1"
+      />
+    </div>
 
-<h2>Editing Instructions</h2>
+    <div class="movie tooltip">
+      <div class="tooltiptext">
+        <h3>Spiderman: Into the Spiderverse</h3>
```

Comment:
despite being an example, there's probably some way to have it so we're not repeating this over and over - know it'll be different when we're pulling actual data - thinking django call, construct html, spit out or something? i am merely spitballing into the void.

maybe a good exercise would be making this be constructed on the backend? like a loop that just makes it five times, spits it out, and then that's rendered on the frontend? then the future functionality just needs to be a server call and then iterating through those instead... hm.

Reply:
Yeah, my next step is going to be to format these for the actual data so I can look into doing that. Julius and I should have the data ready sometime soon.